src/effects/Glitch.tsx

```diff
@@ -22,7 +23,7 @@ export const Glitch = forwardRef<GlitchEffect, GlitchProps>(function Glitch(
   const duration = useVector2(props, 'duration')
   const strength = useVector2(props, 'strength')
```

Comment:
Why don't we use the useVector2 util?
```suggestion
  const strength = useVector2(props, 'strength')
  const chromaticAberrationOffset = useVector2(props, 'chromaticAberrationOffset')
```

Reply:
It must only be a Vector2.

Reply:
Maybe I'm misunderstanding the `useVector2` func, but it looks like it will always return a Vector2:
```
export const useVector2 = (props: Record<string, unknown>, key: string): THREE.Vector2 => {
```

Reply:
It does, but I'm letting R3F set it instead. Maybe I should continue upstream, we shouldn't have to have utils like that.

Reply:
Ahh okay I think I understand now. 
Should we update the docs then? Since the prop `chromaticAberrationOffset` is not being set via props with this change

Reply:
No, this is changing:

```js
class Foo {
  prop = null
}
<foo prop={[0, 0]} /> // error, set of undefined
```

to

```js
class Foo {
  prop = new Vector2()
}
<foo prop={[0, 0]} />
```

I'm preferring for this to be statically known without any work from react-postprocessing so we can benefit from HMR, but I think this can only be done with a code change in postprocessing. In react-postprocessing, everything has the same effect.

Reply:
Thanks for the explanation btw.